### ISSUE TITLE
Resize issues

### ISSUE THREAD

**Issue by runkaiz** [runkaiz/untitled-projects#13]
On mobile devices, the Notes page has its side bar hidden. The reveal button in the top bar only reveals the main menu. While the currently disabled (and useless) button might come in handy for that purpose.

**Comment by yych42** [comment]
The notes page isn't working at all right now. Is it related to #12 ?

**Comment by runkaiz** [comment]
Weird, was working before, let me check it out.

The page has an error of: 
```
500Cannot read property 'isAdmin' of nullTypeError: Cannot read pro...
```

To be honest the error reporting capability of svelte is rather limiting. It usually doesn't point in the right direction but I will look into it anyways. Seems like something went wrong right around the queries.

**Comment by runkaiz** [comment]
> The notes page isn't working at all right now. Is it related to #12 ?

Just checked all the deployments before my latest ones and they all have the same issue, something went wrong on the planetscale side maybe?

**Comment by yych42** [comment]
PS seems to be working fine. I will dig into this.

**Comment by runkaiz** [comment]
Forgot to close this, the issue was fixed in #26.
